packages/components/src/InputGroup/InputGroup.css

```diff
@@ -0,0 +1,61 @@
+.vertical > *:not(:last-child) {
+  margin-bottom: -1px !important;
+}
+
+.horizontal {
+  display: flex;
+  align-items: flex-end;
+}
+
+.horizontal > *:not(:last-child) {
+  margin-bottom: auto;
+  margin-right: -1px;
+}
+
+/* Border Radius Tweak  */
+
+.inputGroup input,
+.inputGroup textarea,
+.inputGroup select {
+  border-radius: 0;
+}
+
+.vertical > :first-child > input,
```

Comment:
This may get more complicated if we support new inputs that aren't true simple html types. For example a color input… This is also something that is more exciting for Formfield as well currently.

I'm not sure that we should worry about it now but it is something we should start thinking about…